docs/source/setup/ecosystem.rst

```diff
@@ -0,0 +1,26 @@
+Isaac Lab Ecosystem
+===================
+
+Isaac Lab is built on top of Isaac Sim to provide a unified and flexible framework
```

Comment:
What about the discussion here?
> https://isaac-sim.github.io/IsaacLab/main/source/setup/faq.html

Seems some duplication?

Reply:
I copied over a bit of the content from FAQ and linked to the page at the bottom. The ask was to make it easier for people to find the information and present it in a more condensed and easier to understand way.